### PR TITLE
Release notes for Percona Server 5.7.14-7

### PR DESCRIPTION
**Pull request by hrvojem**

### CODE REVIEW

File: doc/source/management/audit_log_plugin.rst [hunk 1]
```diff
@@ -356,6 +356,71 @@ or back, first reset the currently-active list to ``NULL``:
   the statements executed within the procedure have the same type
   ``call_procedure`` as well.
 
+.. _filtering_by_database:
+
+Filtering by database
+=====================
+
+In :rn:`5.7.14-7` |Percona Server| has implemented filtering by SQL command
```
Comment (laurynas-biveinis) [inline doc/source/management/audit_log_plugin.rst]: says "SQL command type"

File: doc/source/management/audit_log_plugin.rst [hunk 2]
```diff
@@ -356,6 +356,71 @@ or back, first reset the currently-active list to ``NULL``:
...
+type. This was implemented by adding two new global variables:
+:variable:`audit_log_include_databases` and
+:variable:`audit_log_exclude_databases` to specify which command types should be
```
Comment (laurynas-biveinis) [inline doc/source/management/audit_log_plugin.rst]: x2

File: doc/source/management/audit_log_plugin.rst [hunk 3]
```diff
@@ -356,6 +356,71 @@ or back, first reset the currently-active list to ``NULL``:
...
+
+.. warning:: 
+
+  Only one of these variables can contain a list of databases to be either
+  included or excluded, while the other needs to be ``NULL``. If one of
+  the variables is set to be not ``NULL`` (contains a list of databases),
+  the attempt to set another one will fail. Empty string means an empty list.
+
+
+If query accessing any of databases listed in
```
Comment (laurynas-biveinis) [inline doc/source/management/audit_log_plugin.rst]: s/accessing/is accessing

File: doc/source/management/audit_log_plugin.rst [hunk 4]
```diff
@@ -356,6 +356,71 @@ or back, first reset the currently-active list to ``NULL``:
...
+:variable:`audit_log_include_databases`, the query will be logged.
+If query accessing only databases listed in
```
Comment (laurynas-biveinis) [inline doc/source/management/audit_log_plugin.rst]: Likewise

File: doc/source/management/audit_log_plugin.rst [hunk 5]
```diff
@@ -356,6 +356,71 @@ or back, first reset the currently-active list to ``NULL``:
...
+:variable:`audit_log_exclude_databases`, the query will not be logged.
+:ref:`audit_log_plugin` doesn't receive table access notification for queries
```
Comment (laurynas-biveinis) [inline doc/source/management/audit_log_plugin.rst]: CREATE TABLE statements are logged unconditionally

File: doc/source/release-notes/Percona-Server-5.7.14-7.rst [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+
+ Due to security reasons ``ld_preload`` libraries can now only be loaded from
+ the system directories (:file:`/usr/lib64`, :file:`/usr/lib`) and the *MySQL*
+ installation base directory.
+
+ In the client library, any EINTR received during network I/O was not handled
+ correctly. Bug fixed :bug:`1591202` (upstream :mysqlbug:`82019`).
+
+ ``SHOW GLOBAL STATUS`` was locking more than the upstream implementation which
+ made it unsuitable to be called with high frequency. Bug fixed :bug:`1592290`.
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.7.14-7.rst]: s/unsuitable/less suitable

File: doc/source/release-notes/Percona-Server-5.7.14-7.rst [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+ percona-server source would be missing the spec file required to build the
+ RPMs. Bug fixed :bug:`1600051`.
+
+ :ref:`audit_log_plugin` did not transcode queries. Bug fixed :bug:`1602986`.
+
+ If the changed page bitmap redo log tracking thread stops due to any reason,
+ then shutdown will wait for a long time for the log tracker thread to quit,
+ which it never does. Bug fixed :bug:`1606821`.
+
+ Changed page tracking was initialized too late by |InnoDB| in crash recovery.
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.7.14-7.rst]: too late by InnoDB.

**Comment by laurynas-biveinis** [comment]
Non-TokuDB parts LGTM

**Comment by george-lorch** [comment]
+1 on TokuDB

**Pull request merged**